cli/.jvm/src/main/scala/aqua/PlatformUtils.scala

```diff
@@ -0,0 +1,12 @@
+package aqua
+
+import fs2.io.file.Path
+import scribe.Logging
+
+import scala.util.Try
+
+// Java-specific functions
+object PlatformUtils extends Logging {
+  // you need to add import with builtin.aqua manually when running in JVM environment
```

Comment:
Are you adding builtins to `aqua run` (which is present only in JS and is completely meaningless in JVM environment) or to all the builds as automatic imports?